.github/workflows/docker.build.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         uses: crazy-max/ghaction-docker-meta@v4.0.1
         with:
           images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=sha
```

Comment:
Do we want to tag all things that trigger this job?
Note this means unmerged un reviewed pull requests.

I imagine it might be cleaner to keep the tagged images in the image repository as merged into main code?

Reply:
I might be misunderstanding but this doesn't tag and push images built from random PRs: sure they get tagged but they are never pushed to ghcr (see: https://github.com/wbstack/ui/pull/536/files#diff-c59f4f9a90c4b34cef2c040c881f906daad13fde73566f0e34954f46b0f6efa3R57)